tox.ini

```
@@ -3,7 +3,8 @@ skipsdist = False
 usedevelop = true
 envlist =
     py{36,37,38,39}-dj{22,32}
-    py{38,39}-dj{40,405}
+    py{38,39,310}-dj{40,405}
+	py311-dj42
```

Django 4.2 supports still Python 3.8
```suggestion
 py{38,39,310,311}-dj{40,405,42}
```

---

accepted change